Makefile

```diff
@@ -54,6 +54,8 @@ run: | build run-kernel-gateway
 		-p $(HTTP_PORT):$(HTTP_PORT) \
 		-p $(HTTPS_PORT):$(HTTPS_PORT) \
 		-p 9711:8080 \
+		-e USERNAME=$(USERNAME) \
+		-e PASSWORD=$(PASSWORD) \
```

Comment:
If not assigned this will set to auth with blank strings.  Either check that both are not blank too or remove from the run target.